### ISSUE TITLE
Extended Crew Requirements

### ISSUE THREAD

**Issue by KceGio** [popstarfreas/DG-TS#82]
**Is your feature request related to a problem? Please describe.**
Due to the recent griefing in build dimension, it has prompted me to suggest stricter requirements for Crew.

**Describe the solution you'd like**
Members that are already crew will not have their crew denied if they don't meet the requirements. New crew requirements include, 
- Being at least level 30 on DTP.
- Having an item of r7+.
- Having at least 1 credit(credit won't be consumed, this makes sure that they've at least voted 20 times or donated a dollar).
- Having discord connected to their account.

**Comment by popstarfreas** [comment]
I don't think this is necessary. How are the current requirements not good enough?

**Comment by KceGio** [comment]
I don't think the current requirements are not good enough, my additional conditions ensure that other players can have a sense of trust in the player trying to get crew.
